force-app/main/default/lwc/jsconfig.json

```diff
@@ -22,7 +22,8 @@
     },
     "include": [
         "**/*",
-        "../../.sfdx/typings/lwc/**/*.d.ts"
+        "../../.sfdx/typings/lwc/**/*.d.ts",
+        "../../../../.sfdx/typings/lwc/**/*.d.ts"
```

Comment:
Is this on purpose?

Reply:
@xinnanlisf , nope. Removed this update.